react-components/src/architecture/concrete/primitives/box/BoxView.ts

```diff
@@ -68,7 +68,7 @@ export class BoxView extends GroupThreeView {
   // ==================================================
 
   public override get domainObject(): BoxDomainObject {
-    return super.domainObject as BoxDomainObject;
+    return super.domainObject;
```

Comment:
Is it not better to call the domainObject as his inheritance instead of calling direct from the super class? Or doesnt work in this case for some reason that I'm not following?